docs/EN_US/ECLLanguageReference/ECLR_mods/Recrd-DATASET.xml

```diff
@@ -294,6 +294,16 @@
           <entry>Optional. Specifies records are created on every
           node.</entry>
         </row>
+
+        <row>
+          <entry><emphasis role="bold">__OPTION__(LEGACY)</emphasis></entry>
+
+          <entry>Optional. Specifies that dataset contains deprecated record
```

Comment:
I'm not sure this is quite right.  More like:

Specifies that the dataset should be read using the legacy mechanism, which does not support remote field projection.  This may be required if the dataset contains a combination of alien data types and virtual fields.